EvidenceApi/V1/UseCase/GetEvidenceRequestsUseCase.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using EvidenceApi.V1.Boundary.Response;
+using EvidenceApi.V1.Boundary.Response.Exceptions;
+using EvidenceApi.V1.Factories;
+using EvidenceApi.V1.Gateways.Interfaces;
+using EvidenceApi.V1.Domain;
+using System.Collections.Generic;
+using EvidenceApi.V1.UseCase.Interfaces;
+using EvidenceApi.V1.Boundary.Request;
+
+namespace EvidenceApi.V1.UseCase
+{
+    public class GetEvidenceRequestsUseCase : IGetEvidenceRequestsUseCase
```

Comment:
This could be named better, to differentiate from Find.

Maybe we can do `FindEvidenceRequestByIDUseCase` and `FindEvidenceRequestsUseCase`, or instead use `Search` somewhere in the name of this use case.